docroot/themes/humsci/humsci_basic/src/scss/elements/_base.scss

```diff
@@ -109,34 +109,3 @@ button:not([class]) {
     cursor: pointer;
   }
 }
-
-// Hide the FontAwesome icon that is configured by default in
-// config/default/extlink.settings.yml
-// and replace with different icon
-.fa-ext {
-  // scss-lint:disable ImportantRule
-  margin-right: 0 !important; // removes link underline from icon
-  // scss-lint:enable ImportantRule
-
-  @include hb-colorful {
-    @include hb-external-link-icon;
-
-    .fa-arrow-right {
-      display: none;
-    }
-  }
-}
-
-// Update the base of all regions to use 1.6 rems.
```

Comment:
looks like this contextual-region style might not exist anymore, might want to check if it should be deleted or not

Reply:
I think this class is used when editing in the layout builder.